docs/next/components/VersionDropdown.tsx

```diff
@@ -59,12 +74,16 @@ export default function VersionDropdown() {
                       <span className="text-sm font-medium leading-5 text-gray-900">
                         {currentVersion}
                       </span>
-                      . You can select a different version below.
+                      . You can select a different version below. Note that prior to 1.0, all
```

Comment:
A few thoughts here:

- It feels a bit odd to have this copy display in its entirety for versions prior to 1.0 - to me it seems unnecessary
- It's also a bit long for its current location. Perhaps we could have an icon with a tooltip (or something) display when a version with diverging lib versions is selected
   We could link out to the page that explains our versioning methodology, too: `/getting-started/releases#dagster-integration-libraries`

Reply:
- The copy only displays when you click the menu, which is the same regardless of the version you're browsing, right? It's meant to explain why some entries show two versions and some show one version. If you are on an older version and browsing the list of all versions (including the new ones), you want to know this information.
- I agree it's a little long, but I don't see how to compress it while still explaining what must be explained. I attempted a tooltip on hover over the dropdown, but could not find a pre-existing tooltip component in the docs site (or Headless UI framework) and didn't have the time to spend creating a custom one now (I don't think there are tooltips elsewhere in docs).